04-ruby-mvc-sinatra/sinatra-rest/readme.md

```diff
@@ -198,24 +267,55 @@ Let's compare the output of GET requests initiated in Postman and in the browser
 
 ## We do: Forms
 
-Forms with a POST action are useful for creating new things.
+Forms with a POST action are used for creating new things.
```

Comment:
maybe mention here that forms can only be GET or POST. this is a limitation of HTML, not REST.